examples/sycl/pvc/flash_attention_v2/pvc_fmha_fwd_ruuner.hpp

```diff
@@ -440,18 +417,10 @@ int main(int argc, const char** argv)
   using ElementInputKV = bfloat16_t;                        // <- data type of elements in input matrix B
   using ElementOutput = float;                        // <- data type of elements in output matrix D
 
-  using LayoutQ = cutlass::layout::RowMajor;
-  using LayoutK = cutlass::layout::RowMajor;
-  using LayoutV = cutlass::layout::RowMajor;
-  using LayoutO = cutlass::layout::RowMajor;
-  using LayoutLSE = cutlass::layout::RowMajor;
-
-  // Workgroup-level tile
-  using TileShape = Shape<_128, _64, _32>;
 
   using TiledMma =
       TiledMMA<MMA_Atom<XE_8x16x16_F32BF16BF16F32_TT>,
-               Layout<Shape<_4, _2, _1>, Stride<_2, _1, _0>>,
+               Layout<SubgroupShape, Stride<_2, _1, _0>>,
```

Comment:
Is the `Stride<2,1,0>` safe with e.g. SubgroupShape `<8, 1, 1>`?

Reply:
Yes fixed